Tasks/AndroidSigning/task.json

```diff
@@ -81,6 +84,7 @@
         },
         {
             "name": "keyPass",
+            "aliases": ["jarsignKeyPassword"],
```

Comment:
The prefix here seems to have changed from jarsigner to jarsign. Seems like we should use a consistent prefix.